src/Binary/Loader/AbstractDoctrineLoader.php

```diff
@@ -22,16 +22,12 @@ abstract class AbstractDoctrineLoader implements LoaderInterface
      */
     protected $manager;
 
-    /**
-     * @var string
-     */
-    protected $class;
+    protected string $class;
 
     /**
      * @param ObjectManager|LegacyObjectManager $manager
-     * @param string                            $class
      */
-    public function __construct($manager, $class = null)
+    public function __construct($manager, string $class)
```

Comment:
this parameter used to be optional. would the code later fail if we pass null? and what class is this? if it is the class of the entity, we could name the parameter $entityClass?

Reply:
I've changed based on the `ObjectManager::find` call:

https://github.com/doctrine/persistence/blob/1ed5427e2ea50adb17aa6bb40037a8aa6e99a9fe/lib/Doctrine/Persistence/ObjectManager.php#L11-L27

It could be also a document I guess, I've changed to $modelClass